tcr-engine/toolchain/toolchain.go

```diff
@@ -47,7 +47,7 @@ type (
 		GetBuildCommands() []Command
 		GetTestCommands() []Command
 		RunBuild() error
-		RunTests() error
+		RunTests() (string, error)
```

Comment:
If we move extraction from test output to test run info struct into toolchain, we could then change the RunTests() signature so that it returns a struct with test run info instead of the entire test trace